.github/workflows/all.yml

```diff
@@ -0,0 +1,319 @@
+name: Deps, Lint, Test, Build, Publish
```

Comment:
I think we should change this to just `CI`. This looks a bit ghastly :sweat_smile: 
![Screenshot_2020-10-08 Replace CircleCI AppVeyor with GitHub Actions by imiric · Pull Request #1640 · loadimpact k6](https://user-images.githubusercontent.com/3336773/95466066-97ad9280-0984-11eb-9c44-de36b9a2e859.png)
.

Reply:
😄 Sure. What's really annoying is that [you can't (easily) have dependencies between jobs in separate workflows](https://github.community/t/how-do-i-specify-job-dependency-running-in-another-workflow/16482/11)... Ideally, there should be a `publish.yml` workflow that only runs if, say, the `test.yml` and `build.yml` workflows passed. That would've cleaned up this long name and avoided having a huge single `all.yml` workflow.

Reply:
Resolved in 8a6c78da.